models/staging/edfi_3/stage/stg_ef3__contacts.sql

```diff
@@ -0,0 +1,37 @@
+with base_contacts as (
+    select * from {{ ref('base_ef3__contacts') }}
+    where not is_deleted
+),
+base_parents as (
+    select * rename parent_unique_id as contact_unique_id
+    from {{ ref('base_ef3__parents') }}
+    where not is_deleted
+),
+-- parents were renamed to contacts in Data Standard v5.0
+unioned as (
+    select * from base_contacts
+    union 
+    select * from base_parents
+),
+keyed as (
+    select 
+        {{ dbt_utils.surrogate_key(
+            [
+                'tenant_code',
+                'lower(contact_unique_id)'
+            ]
+        ) }} as k_contact,
+        unioned.*
+        {{ extract_extension(model_name=this.name, flatten=True) }}
+    from unioned
+),
+deduped as (
+    {{
+        dbt_utils.deduplicate(
```

Comment:
this looks correct but might be nice to test this out in SC to see edge case behavior where there's overlapping parents & contacts (if there are any)

Reply:
SC didn't have any overlapping parents and contacts because they made a change to remove a leading zero from their IDs. I think ideally they would be the same to avoid duplicates in dim_parent